src/data/breaking-things-data.ts

```diff
@@ -0,0 +1,56 @@
+import { BreakACDifficultyTypeGroup, BreakHPDifficultyTypeGroup } from '../types/dc-types';
+
+// data
+export const OBJECT_BREAK_HP_INFO: BreakHPDifficultyTypeGroup = [
+    {
+        text: 'Tiny (bottle, lock)',
+        fragile: '2 (1d4)',
+        resilient: '5 (2d4)',
+    },
+    {
+        text: 'Small (chest, lute)',
+        fragile: '3 (1d6)',
+        resilient: '10 (3d6)',
+    },
+    {
+        text: 'Medium (barrel, chandelier)',
+        fragile: '4 (1d8)',
+        resilient: '18 (4d8)',
```

Comment:
I would rename `fragile` to `fragility` and `resilient` to `resiliency` -- maybe?

I'm wondering if we should break the "die" into its own type? So you could supply something like:

```
{
    text: Medium (barrel, chandlier),
    fragility: {
        avg: 4
        die: {
            num: 1
            sides: 8
        }
     }
    resiliency: {
        avg: 18
        die: {
            num: 4
            sides: 8
        }
     }
}
```

This way, later (in a future feature) we could supply the `dice` object to a click function that would do the roll for the user?

Reply:
They were named with the idea that a 
'fragile' 'Medium' item had this hp
or a 'resilient' 'Large' item had this hp.

Re: Dice
I hadn't considered adding them as dice. I guess I didn't envision having rolling capabilities in this. I only envisioned this as supplemental reference.

Do you want this functionality? 
If so, do you suggest re-structuring the type now or as part of the issue with this future feature?

Reply:
The naming isn't super important -- this is just a suggestion.

Whether or not we add rolling functions later, I think we should consider how this data could be used in the future in combination with other data on the page or in combination with other data. We could do things like: show/hide dice values or show/hiding the averages with a click, creating a "Die" component that shows an image based on the supplied die base (ie. 1d8 shows an image of a d8, 2d4 + 1d6 shows two d4 images and a d6 image).

Ultimately, if we're using these data objects to only create key -> string pairs we should maybe reconsider how we're building out some of these pages using data objects as this may be more effort rather than using Markdown files and simple HTML lists.

Reply:
addressed in 1ca2df8